content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Let's celebrate Java 11 Support on Jenkins"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: alecharp
+links:
+  gitter: /jenkinsci/platform-sig
+  googlegroup: jenkins-platform-sig
+  sig: platform
+---
+
+NOTE: This is a joint blogpost prepared by the Java 11 Support Team: Adrien Lecharpentier, Baptiste Mathus, Kevin Earl, Mark Waite, Ramon Leon, Oleg Nenashev, Maria Isabel Vilacides.
+
+image:/images/logos/formal/256.png[Jenkins Java, role=center, float=right]
+
+We have worked hard for this and it's now here.
+We are happy to announce the **full support of Jenkins over Java 11**.
+This means you can now run your instance on with a Java 11 JVM.
```

Comment:
You just need "with a Java 11 JVM", not "on with a Java 11 JVM"

Reply:
```suggestion
This means you can now run your instance with a Java 11 JVM.
```